.github/workflows/ci.yml

```
@@ -105,6 +105,7 @@ jobs:
           storybookBuildDir: storybook-static
           autoAcceptChanges: main
           exitOnceUploaded: true
+          onlychanged: true
```

```suggestion
          onlyChanged: true
```
Docs say this is supposed to be onlyChanged, I'm surprised it works 😅